Configs/Require.plist

```diff
@@ -15,7 +15,7 @@
 	<key>CFBundlePackageType</key>
 	<string>FMWK</string>
 	<key>CFBundleShortVersionString</key>
-	<string>1.0</string>
+	<string>3.0.0</string>
```

Comment:
I added Swift 4.2 compatibility to my fork a while back and introduced this release for my own purposes. John's fork is at release 2.1 at moment so 3.0.0 seemed like a good bump.

Haven't bumped again to 3.1.0 because it's fork specific.